### PR TITLE
Moving templates and such

### PR DESCRIPTION
**Pull request by SunnevaThorsteins**
Templates added to a separate documents, contribution guidelines added and names and emails of current contributors.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ This is work in progress.
      <li><a href="#goals">Goals</a></li>
      <li><a href="#contributors">Contributors</a></li>
      <li><a href="#deliverables">Deliverables</a></li>
+     <li><a href="#contributors">Contributors>/a><li>
```
Comment (judyfong) [inline readme.md:17]:
```suggestion
     <li><a href="#contributors">Contributors</a><li>
```

File: readme.md [hunk 2]
```diff
@@ -187,6 +188,17 @@ Translating from the SÍM requirements to these deliverables will need to be dis
...
+#### Names and emails of contributors 
+
+* Haukur Páll Jónsson <haukurpj@ru.is>
+* Judy Yum Fong <judyfong@ru.is>
+* Ólafur Helgi Jónsson <olafurhj@ru.is>
+* Sunneva Þorsteinsdóttir <sunnevath@ru.is>
+* Þorsteinn Daði Gunnarsson <thorsteinng@ru.is>
+
+### Contribution guidelines
+To contribute to this project please submit a pull request to the master branch and request a review from someone in the software guidelines team. If you have a lot of suggestions feel to make multiple requests.
```
Comment (judyfong) [inline readme.md:200]:
```suggestion
To contribute to this project please submit a pull request to the master branch and request a review from someone in the software guidelines team. If you have a lot of suggestions feel free to make multiple requests. 
```

File: readme.md [hunk 3]
```diff
@@ -14,6 +14,8 @@ This is work in progress.
      <li><a href="#goals">Goals</a></li>
      <li><a href="#contributors">Contributors</a></li>
```
Comment (judyfong) [inline readme.md:15]: this line should be deleted

Comment (SunnevaThorsteins) [reply readme.md:15]: I'm not sure what you mean by this, I fail to find this line in the readme.md file

Comment (HaukurPall) [reply readme.md:15]: I also think this should be "maintainers" rather than "contributors". We should also add a section about how we accept/decline PRs.

Comment (judyfong) [inline readme.md:15]:
```suggestion
```

Comment (judyfong) [reply readme.md:15]: This is what I meant, @SunnevaThorsteins.

File: readme.md (hunk 1 above)
Comment (judyfong) [inline readme.md]: I believe this is the line that @HaukurPall would want changed to "Maintainers" and also within the file. I agree that maintainers would be better in the long run.

File: readme.md (hunk 3 above)
Comment (judyfong) [reply readme.md:15]: This change is resolved as far as I am concerned. Although there is also the change that @HaukurPall suggested about adding how we plan to accept/decline PRs. So I'll leave it to him to mark this as resolved if he agrees.

**Review by HaukurPall — APPROVED** [review]

File: readme.md (hunk 1 above)
Comment (HaukurPall) [reply readme.md]: I wlil fix this in a subsequent PR.

**Pull request merged**